0000-template.md

```diff
@@ -1,15 +1,22 @@
 # Feature name
 
 * Proposal: [SE-NNNN](NNNN-filename.md)
-* Author: [Swift Developer](https://github.com/swiftdev)
+* Authors: [Author 1](https://github.com/swiftdev), [Author 2](https://github.com/swiftdev)
+* Review Manager: TBD
 * Status: **Awaiting review**
-* Review manager: TBD
+
+*During the review process, add the following fields as needed:*
+
+* Decision Notes: [Rationale](https://lists.swift.org/pipermail/swift-evolution/), [Additional Commentary](https://lists.swift.org/pipermail/swift-evolution/)
+* Bugs: [SR-NNNN](https://bugs.swift.org/browse/SR-NNNN), [SR-MMMM](https://bugs.swift.org/browse/SR-MMMM)
```

Comment:
A single bug report is expected. The only current exceptions are [SE-0007](https://github.com/apple/swift-evolution/blob/master/proposals/0007-remove-c-style-for-loops.md) and [SE-0061](https://github.com/apple/swift-evolution/blob/master/proposals/0061-autoreleasepool-signature.md).